data-prepper-logstash-configuration/build.gradle

```diff
@@ -11,6 +11,9 @@ repositories {
 dependencies {
     antlr "org.antlr:antlr4:4.9.2"
     implementation project(':data-prepper-api')
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
+    implementation 'com.fasterxml.jackson.core:jackson-databind'
+    implementation 'org.slf4j:slf4j-simple:1.7.32'
```

Comment:
This should be `testImplementation` if we use it all. Otherwise, it goes into the main classpath, but there should be only one SLF4J in the classpath.

Reply:
Without this there's a warning and it's not showing the Log.warn message in log
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.